content/first-steps.md

```diff
@@ -100,3 +100,10 @@ $ npm run start
 ```
 
 This command starts the app with the HTTP server listening on the port defined in the `src/main.ts` file. Once the application is running, open your browser and navigate to `http://localhost:3000/`. You should see the `Hello World!` message.
+
+To watch for changes in your files, you can run the following command to start the application:
+```bash
+$ npm run start:dev
+```
+
+This command will watch your files for changes and automatically recompile and reload the server.
```

Comment:
```suggestion
This command will watch your files and automatically recompile and reload the server.
```

To avoid saying "for changes" twice. What do you think @jmcdo29 ?

Reply:
Ok cool!

Reply:
Isn't it better maybe to do this:

"This command will observe your files, automatically recompiling and reloading the server"

so as not to repeat the "and" twice

What do you think?